packages/fxa-auth-server/test/local/routes/subscriptions.js

```diff
@@ -528,4 +528,124 @@ describe('subscriptions', () => {
       }
     });
   });
+
+  describe('POST /oauth/subscriptions/reactivate', () => {
+    it('should reactivate cancelled subscriptions', async () => {
+      const res = await runTest(
+        '/oauth/subscriptions/reactivate',
+        {
+          ...requestOptions,
+          method: 'POST',
+          payload: { subscriptionId: SUBSCRIPTION_ID_1 },
+        }
+      );
+
+      assert.equal(customs.check.callCount, 1);
+
+      assert.equal(db.reactivateAccountSubscription.callCount, 1);
+      let args = db.reactivateAccountSubscription.args[0];
```

Comment:
Should we assert `subhub.reactivateSubscription` was called?

Reply:
Oh, we should yeah. I merged this before I saw your comment, sorry.

Reply:
No worries, forgot to hit the submit review button 🤦🏽‍♂️